calico_containers/tests/st/test_mainline.py

```diff
@@ -42,15 +37,8 @@ def run_mainline(self, ip1, ip2):
         node1_pid = host.execute("docker inspect --format '{{.State.Pid}}' node1").stdout.rstrip()
         node2_pid = host.execute("docker inspect --format '{{.State.Pid}}' node2").stdout.rstrip()
 
-        for i in range(10):
-            try:
-                host.execute("./nsenter -t %s ping %s -c 1 -W 1" % (node1_pid, node2_ip))
-                break
-            except ErrorReturnCode:
-                if i == 9:
-                    raise AssertionError("Network failed to come up.")
-                else:
-                    sleep(1)
+        ping = partial(host.execute, "./nsenter -t %s ping %s -c 1 -W 1" % (node1_pid, node2_ip))
```

Comment:
I don't really know why Rob added `nsenter`. I'll consider this if/when I get to writing an `assert_can_ping`.